### ISSUE TITLE
Rule proposal: Enforce empty line below imports

### ISSUE THREAD

**Issue by sindresorhus** [benmosher/eslint-plugin-import#244]
I'm aware this is purely stylistic, but thought I'd propose it since it seems this plugin will soon expand to stylistic rules with the adoption of https://github.com/jfmengels/eslint-plugin-import-order

## Fail

```js
import foo from 'foo';
import bar from 'bar';
const unicorn = true;
```

```js
const foo = require('foo');
const bar = require('bar');
const unicorn = true;
```

## Pass

```js
import foo from 'foo';
import bar from 'bar';

const unicorn = true;
```

```js
const foo = require('foo');
const bar = require('bar');

const unicorn = true;
```

-

// @jamestalmage @jfmengels

**Comment by benmosher** [comment]
Sounds good to me, will accept a PR.

**Comment by benmosher** [comment]
Oh weird, expected #245 to close this but I guess it didn't mention it explicitly.
